spec/rails_helper.rb

```diff
@@ -1,10 +1,15 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
-require_relative '../config/environment'
+# require_relative '../config/environment'
+
+require File.expand_path('../config/environment', __dir__)
 # Prevent database truncation if the environment is production
 abort("The Rails environment is running in production mode!") if Rails.env.production?
 require 'rspec/rails'
+require 'capybara/rspec'
+
+Capybara.default_driver = :selenium_chrome
```

Comment:
- [ ] As for this project, you were requested to add integration specs for views, however in your specs' folder there is no written specs for the view, Kindly add the specs of all the feature you may have integrated in your project as required 👍

![Screenshot 2023-11-21 121525](https://github.com/ishimwepati/Blog-Rails/assets/80458199/0f28c2f4-efb8-43ac-928f-81e2943665ab)

<img width="452" alt="Screenshot 2023-11-21 121552" src="https://github.com/ishimwepati/Blog-Rails/assets/80458199/d6a4ff78-080f-42bf-90e6-4a60ef49c6eb">

Reply:
Thanks a lot 👍 !